Command/CreateSiteCommand.php

```diff
@@ -80,7 +80,24 @@ public function execute(InputInterface $input, OutputInterface $output)
             $values[$name] = $input->getOption($name);
 
             while ($values[$name] == null) {
-                $values[$name] = $helper->ask($output, sprintf('Please define a value for <info>Site.%s</info> : ', $name));
+                /*
+                 * NEXT_MAJOR: remove if statement and use the question helper only (true value of if) when dropping sf < 2.5
+                 */
+                if ($this->getHelperSet()->has('question')) {
+                    $question = new \Symfony\Component\Console\Question\Question(
```

Comment:
strange i couldnt reply to your previous comment, but tested both 2.3 and 3.2.8. it slipped through the cracks previously because i was using the arguments in the command line but now i properly tested with and without arguments.

i've used both the question and the confirmation question from previous pull request so should be nice and neat now